.gitignore

```diff
@@ -34,7 +34,8 @@ venv
 # Autogenerated Protobufs
 lib/streamlit/proto/*_pb2.py
 lib/streamlit/proto/*_pb2.pyi
-frontend/src/autogen
+frontend/src/lib/proto.js
```

Comment:
Right now, this works because it's in CRA. However, this won't work when we actually do the split. Should we just make it so we copy it to both folders and have them consume different protobufs?

Reply:
I don't follow - why won't this work if we stop using CRA (or CRACO)?

Reply:
Actually, sorry! I thought we had to duplicate the protobufs based on my past PR but that is definitely wrong 😅 . Resolving this.